lib/Styliner.js

```diff
@@ -124,6 +123,19 @@ function parseDataUri(uri) {
 		return decodeURIComponent(parsed[3]);
 }
 
+/**
+ * Asynchronously parses an HTML document from file path passed as argument and inlines styles as appropriate.
+ *
+ * @param {String}			filePath				The file path to parse HTML document
+ */
+Styliner.prototype.parseFile = function(filePath) {
+  var _this = this;
+  return qfs.read(qfs.join(this.baseDir, filePath)).then(function(content) {
+    return _this.processHTML(content);
```

Comment:
Pass the directory containing the file (`qfs.directory()`), so that relative paths work correctly.